addons/html_editor/static/tests/font_awesome.test.js

```diff
@@ -515,6 +515,31 @@ describe("FontAwesome insertion", () => {
             contentAfter: '<p>abs<i class="fa fa-star"></i>[]cd</p>',
         });
     });
+
+    test("should insert fontAwesome consecutively", async () => {
+        await testEditor({
+            contentBefore: "<p>[]<br></p>",
+            stepFunction: async (editor) => {
+                editor.dispatch("INSERT_FONT_AWESOME", { faClass: "fa fa-star" });
+                editor.dispatch("INSERT_FONT_AWESOME", { faClass: "fa fa-glass" });
+            },
+            contentAfterEdit:
+                '<p><i class="fa fa-star" contenteditable="false">\u200b</i><i class="fa fa-glass" contenteditable="false">\u200b</i>[]</p>',
+            contentAfter: '<p><i class="fa fa-star"></i><i class="fa fa-glass"></i>[]</p>',
+        });
+        await testEditor({
+            contentBefore: "<p>test[]</p>",
+            stepFunction: async (editor) => {
+                editor.dispatch("INSERT_FONT_AWESOME", { faClass: "fa fa-star" });
+                editor.dispatch("INSERT_FONT_AWESOME", { faClass: "fa fa-glass" });
+                editor.dispatch("INSERT_FONT_AWESOME", { faClass: "fa fa-plus" });
```

Comment:
You have already added test for adding 2 icons does anything different happen when inserting 3 icons?